rust/src/io/exec/planner.rs

```diff
@@ -212,9 +232,11 @@ impl Planner {
         // Allow sqlparser to parse filter as part of ONE SQL statement.
 
         let ast_expr = parse_sql_filter(filter)?;
-        let expr = self.parse_sql_expr(&ast_expr)?;
+        let expr_res = self.parse_sql_expr(&ast_expr);
+        let expr = expr_res?;
```

Comment:
Was this there just for debugging?
```suggestion
        let expr = self.parse_sql_expr(&ast_expr)?;
```

Reply:
yeah, reverted